chart/values.yaml

```diff
@@ -7,6 +7,10 @@ version:
 hostname: localhost
 certificatesSecret:
   secretName: https-certificates
+  certName: cert.pem
```

Comment:
This looks like a step backwards from the TLS secret type we're moving to

Reply:
> This looks like a step backwards from the TLS secret type we're moving to

The goal here is to still support the certificates that come out of letsencrypt right away, by default. Otherwise we'd need to tell Self-Hosted users to do yet another step: https://www.gitpod.io/docs/self-hosted/latest/install/configure-ingress#using-lets-encrypt-to-generate-https-certificates